client/my-sites/stats/feedback/index.tsx

```diff
@@ -193,12 +194,13 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 		useNoticeVisibilityHooks( siteId );
 
 	useEffect( () => {
-		if ( ! isPending && ! isError && shouldShowFeedbackPanel ) {
+		if ( ! isPending && ! isError && shouldShowFeedbackPanel && supportCommercialUse ) {
```

Comment:
It'd better to have the condition outside of the the component itself, because it might change later, or used in different places with different conditions. Plus they controlling logic should happen in one place; otherwise so easy to miss..